packages/block-editor/src/components/link-control/settings-drawer.js

```diff
@@ -44,9 +44,9 @@ const LinkControlSettingsDrawer = ( {
 
 	return (
 		<fieldset className="block-editor-link-control__settings">
-			<legend className="screen-reader-text">
-				{ __( 'Currently selected link settings' ) }
-			</legend>
+			<VisuallyHidden>
```

Comment:
We need `<VisuallyHidden as="legend">` here. A `<legend>` must be the first direct descendant of a `<fieldset>`, so we can't have any intermediate wrappers.